font/src/ft/mod.rs

```diff
@@ -320,14 +344,25 @@ impl FreeTypeRasterizer {
                 glyph_key.size.as_f32_pts() * self.device_pixel_ratio * 96. / 72.
             });
 
-        face.ft_face.set_char_size(to_freetype_26_6(size), 0, 0, 0)?;
+        if !face.has_color {
+            face.ft_face.set_char_size(to_freetype_26_6(size), 0, 0, 0)?;
+        }
 
         unsafe {
             let ft_lib = self.library.raw();
-            freetype::ffi::FT_Library_SetLcdFilter(ft_lib, face.lcd_filter);
+            if !face.has_color {
+                freetype::ffi::FT_Library_SetLcdFilter(ft_lib, face.lcd_filter);
+            }
         }
 
-        face.ft_face.load_glyph(index as u32, face.load_flags)?;
+        let flags = if face.has_color {
+            freetype::face::LoadFlag::COLOR | freetype::face::LoadFlag::DEFAULT
+        } else {
+            face.load_flags
+        };
+
+        face.ft_face.load_glyph(index as u32, flags)?;
+
```

Comment:
I'm fairly sure you don't need to special case the LCD filter and load flags. AFAIK they only apply to outline fonts. Case in point: regular bitmap fonts already work just fine in Alacritty.

Reply:
If you want to render your colored emojis in monochrome variant then go ahead and remove load color. I'll investigate a bit more on what we can do here. I don't know how LCD filter affects, but I've noticed that some implementations disable it for colored glyphs.

Reply:
I know `FT_LOAD_COLOR` is required for color bitmap fonts. That's not what I meant.

Reply:
Do you mean about that I don't need to manually alter them here? If this is what you wanted to say then you're right.

Reply:
From my own experience, there's no need to differentiate load flags **at all**. From what I have seen, you can _always_ include `FT_LOAD__COLOR`, even for non-color bitmap fonts. And for color bitmap fonts it appears safe to include the "normal" load flags too.

I'm afraid the documentation of `FT_LOAD_COLOR` doesn't really back this up. But neither does it say anything against it.

But this is just from my own experience. I know others only add `FT_LOAD_COLOR` for color bitmap fonts.

I'm for simplifying code and avoiding special casing where it isn't needed. This appears to be one such case, but I don't have any hard evidence.